core/src/main/java/dev/streamx/cli/ingestion/publish/payload/InitialPayloadResolver.java

```diff
@@ -0,0 +1,41 @@
+package dev.streamx.cli.ingestion.publish.payload;
+
+import dev.streamx.cli.exception.PayloadException;
+import dev.streamx.cli.ingestion.publish.PayloadArgument;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jetbrains.annotations.NotNull;
+
+@ApplicationScoped
+class InitialPayloadResolver {
+
+  @Inject
+  ValueReplacementExtractor valueReplacementExtractor;
+
+  @NotNull
+  InitialPayload computeInitialPayload(List<PayloadArgument> payloadArguments) {
+    String initialPayload = "{}";
+    List<PayloadArgument> replacements = payloadArguments;
+
+    String firstPayloadArgValue = payloadArguments.stream()
+        .map(PayloadArgument::getValue)
+        .findFirst()
+        .orElseThrow(PayloadException::payloadNotFound);
+
+    boolean firstDataIsInitialPayload = valueReplacementExtractor.extract(firstPayloadArgValue)
+        .map(Pair::getLeft)
```

Comment:
Why do we need this mapping here? If `firstPayloadArgValue` does not contain valid json path then whole object is empty optional

Reply:
basicly - you're right :) 
I'll remove this mapping